### ISSUE TITLE
Test all of WCAG AA with Lightouse and aXe-core

### ISSUE THREAD

**Issue by robdodson** [GoogleChrome/lighthouse#1486]
This should be pretty straightforward. Rather than write a bunch of boilerplate audits I may try to see if I can make a general purpose accessibility audit that all of these axe violations use.

**Comment by jasonday** [comment]
Any update on progress?

**Comment by robdodson** [comment]
oh this went in a long time ago.

**Comment by robdodson** [comment]
here's the rules we support:
https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/gather/gatherers/accessibility.js#L23-L34

wcag2a, wcag2aa, and a few extras that are labeled "best-practice" by axe
